### ISSUE TITLE
This is pretty cool

### ISSUE THREAD

**Issue by tshin7** [BarronFritz/jVQ#1]
I found your github by randomly searching for "github vagabond's quest" on google after I found out that there was a open source version of runescape on github. This is the vagabond's quest that used to be on netdragons right?

**Comment by BarronFritz** [comment]
Yeah. I've been trying to remake/refresh the old netdragons Vagabonds quest+ since the site went down 5 or more years ago. I just haven't had the time to do anything productive on it.

**Comment by tshin7** [comment]
Did you also try to remake it in C++ too? I wonder if there are others who are interested in remaking 
VQ. It'd be cool if we could find a group of people and somehow get this working.
